portfolio/src/main/webapp/index.html

```diff
@@ -7,11 +7,12 @@
     <title>Keith Williams Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAdxLXz6dOxxcT4x9Ymu8DWvN3EsNZwz-8&callback=initMap"></script>
     <script src="https://www.gstatic.com/charts/loader.js"></script>
     <script src="script.js"></script>
 </head>
 
-<body onload = "loadCommentsFromDatastore()">
+<body onload = "createMap()">
```

Comment:
should we execute start() function here?

Reply:
It turns out that functionally, the portfolio operates the same whether start() or createMap() is executed (because when the user selects the number of comments to display, loadCommentsFromDatastore() runs no matter what). I initially thought that I needed start(), but I can remove it.

Reply:
Then let's remove start().